src/mixins/java/org/spongepowered/common/mixin/core/server/level/ServerPlayerMixin.java

```diff
@@ -830,6 +831,30 @@ public void sendMessage(final net.minecraft.network.chat.Component p_241151_1_,
             }
         }
     }
+
+    @Inject(method = "setPlayerInput", at = @At("HEAD"), cancellable = true)
+    public void onSetPlayerInput(final float sway, final float surge, final boolean jumping, final boolean dismount, final CallbackInfo ci) {
+        if(isPassenger()) {
```

Comment:
We prefix all same class accesses with `this.` (but this would become `this.shadow$isPassenger()`)

Also, space after `if`